spec.html

```diff
@@ -6348,7 +6348,7 @@ <h1>GetActiveScriptOrModule ( )</h1>
       <h1>ResolveBinding ( _name_ [ , _env_ ] )</h1>
       <p>The ResolveBinding abstract operation is used to determine the binding of _name_ passed as a String value. The optional argument _env_ can be used to explicitly provide the Lexical Environment that is to be searched for the binding. During execution of ECMAScript code, ResolveBinding is performed using the following algorithm:</p>
       <emu-alg>
-        1. If _env_ is not present or if _env_ is *undefined*, then
+        1. If _env_ is *undefined*, then
```

Comment:
I don't think we can currently assume that "not present" is equivalent to "undefined" for abstract operations.

Reply:
This is correct, however I would be open to some clause 5 changes to make this work for abstract ops. I was also confused by this pattern until Allen explained it to me some years ago.